backend/main.py

```
@@ -41,36 +43,57 @@ async def root():
+            ssh_port=request.ssh_port,
+            target_base_path=request.target_base_path
+        )
+        
         migration_id = await migration_service.start_migration(request)
         return MigrationResponse(
             migration_id=migration_id,
             status="started",
             message="Migration process started successfully"
         )
+    except SecurityValidationError as e:
+        raise HTTPException(status_code=422, detail=f"Security validation failed: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)